common/stringop.c

```diff
@@ -283,6 +293,24 @@ static bool has_whitespace(const char *str) {
 	return false;
 }
 
+
+/*
+ * Returns true if the string contains only whitespace and false otherwise.
+ */
+bool is_empty(const char *str) {
+	bool ret = true;
+	if (str) {
+		while (*str != '\0') {
+			if (!isspace(*(const unsigned char*)str)) {
+				ret = false;
+				break;
```

Comment:
There's no need for the ret variable to exist. During this branch you can just return false directly, and return true at the end, and eliminate the variable.